api/konnect/v1alpha1/konnect_entity_status.go

```diff
@@ -181,3 +181,10 @@ type KonnectEntityStatusWithControlPlaneAndCertificateRefs struct {
 	// CertificateID is the Konnect ID of the Certificate this entity is associated with.
 	CertificateID string `json:"certificateID,omitempty"`
 }
+
+// KonnectEntityStatusWithNetworkRef represents the status of a Konnect entity with reference to a Konnect cloud gateway network.
+type KonnectEntityStatusWithNetworkRef struct {
+	KonnectEntityStatus `json:",inline"`
+	// NetworkID is the Konnect ID of the Konnect cloud gateway network this entity is associated with.
+	NetworkID string `json:"networkID,omitempty"`
```

Comment:
Mark this as optional (if it's `omitempty`).

Reply:
Done.